paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -101,6 +106,11 @@ internal class DefaultFlowController @Inject internal constructor(
     lateinit var flowControllerComponent: FlowControllerComponent
 
     private var paymentLauncher: StripePaymentLauncher? = null
+    private var didLastConfigureFail: Boolean = false
+
+    private val configureRequests = MutableSharedFlow<ConfigureRequest>()
```

Comment:
Discussing out of band. But I think this should be moved to `FlowControllerViewModel`. `DefaultFlowController` is recreated on configuration change. We want this state to persist configurations.